backend/tracim_backend/tests/functional/test_user.py

```diff
@@ -5383,7 +5449,7 @@ def test_api_disable_user__err_403__other_account(self):
         assert 'code' in res.json.keys()
         assert res.json_body['code'] == error.INSUFFICIENT_USER_PROFILE
 
-    def test_api_disable_user__ok_200__user_itself(self):
+    def test_api_disable_user__err_403__user_itself(self):
```

Comment:
`s/403/400/`

Reply:
correct but tricky case, i rename + add comment.

Reply:
the test case is wrong in this case. if the user is a simple user, he cant remove anyone from a workspace - it's the only case to test. There are no special case for himself because he is not allowed to remove any user from the space